src/mca/plm/base/plm_base_launch_support.c

```diff
@@ -302,8 +302,13 @@ void prte_plm_base_setup_job(int fd, short args, void *cbdata)
          * step required before we launch the daemons. It allows
          * the prte_rmaps_base_setup_virtual_machine routine to
          * search all apps for any hosts to be used by the vm
+         *
+         * Note that the prte_plm_base_create_jobid function may/will
+         * have created a temporary job object at this location. It
+         * will also replace that temporary object with the one passed
+         * to it. So this call is redundant.
          */
-        prte_hash_table_set_value_uint32(prte_job_data, caddy->jdata->jobid, caddy->jdata);
```

Comment:
The call to prte_set_job_data_object is commented out. Is there a reason for not updating the hash here?

Reply:
The `prte_plm_base_create_jobid` (just above) will now guarentee that the `caddy->jdata` is in the correct position. This is now redundant. I'll clean up the comment and remove the commented out line of code. I wanted to keep the existing comment in case it is imporant for other parts of the code.